client/app/dedicated/server/bandwidth/cancel-private/dedicated-server-bandwidth-vrack-cancel.controller.js

```diff
@@ -1,6 +1,8 @@
-class ServerCancelBandwidthVrackCtrl {
-  constructor($scope, $stateParams, $rootScope, User, Server, BandwidthVrackOrderService) {
+export default class {
+  /* @ngInject */
+  constructor($scope, $state, $stateParams, $rootScope, User, Server, BandwidthVrackOrderService) {
     this.$scope = $scope;
+    this.$state = $state;
```

Comment:
As we are in a component and use-it in states, we should use `ui-router resolves` instead of injecting `$state` & `$stateParams`

Reply:
I'll refactor a bit those parts